recipes/libcf/build.sh

```diff
@@ -0,0 +1,23 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC ${CFLAGS}"
+export CXXLAGS="${CFLAGS} ${CXXLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export LFLAGS="-fPIC ${LFLAGS}"
+
+if [ "$(uname)" == "Darwin" ]; then
+    export CXXFLAGS="${CXXFLAGS} -fno-common"
+    export MACOSX_DEPLOYMENT_TARGET=$(sw_vers -productVersion | sed -E "s/([0-9]+\.[0-9]+).*/\1/")
+    export DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib
+    export LDFLAGS="${LDFLAGS} -lpython" 
+fi
+
+CONDA_LST=`conda list`
+if [[ ${CONDA_LST}'y' == *'openmpi'* ]]; then
```

Comment:
@ocefpaf @dnadeau4 let's take it out for now. We need to revisit the whole suite eventually to use conda `features` for this. As discussed in earlier conda-forge recipes.